### PR TITLE
Remove some image caches not used by RP for a while to free some VHD space

### PR DESCRIPTION
**Pull request by chengliangli0918**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Some distro VHD content size is greater than 30GB now, so VHD build fails. Remove some image caches not used by RP for a while to free some VHD space.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -95,7 +95,6 @@
     {
       "downloadURL": "mcr.microsoft.com/aks/hcp/hcp-tunnel-front:*",
       "amd64OnlyVersions": [
-        "master.211013.1",
         "master.220527.2"
```
Comment (chengliangli0918) [inline vhdbuilder/packer/components.json:98]: master.220527.2 was used in RP from 4 weeks ago.

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -267,7 +265,6 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/ip-masq-agent:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v2.5.0.11",
         "v2.5.0.12"
```
Comment (chengliangli0918) [inline vhdbuilder/packer/components.json:268]: v2.5.0.12 was used in RP from 2 weeks ago

File: vhdbuilder/packer/components.json [hunk 3]
```diff
@@ -327,11 +324,9 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/open-policy-agent/gatekeeper:*",
       "amd64OnlyVersions": [
-        "v3.4.1",
-        "v3.5.2"
```
Comment (chengliangli0918) [inline vhdbuilder/packer/components.json:331]: v3.5.2 and v3.7.0 was not used in RP for a while.

{{- else if eq .component "open-policy-agent-gatekeeper" -}}
  {{- if semverCompare ">=1.18.0" .version -}}v3.7.1
  {{- else -}}v3.4.1
  {{- end -}}

File: vhdbuilder/packer/components.json [hunk 4]
```diff
@@ -393,8 +388,7 @@
     {
       "downloadURL": "mcr.microsoft.com/azure-policy/policy-kubernetes-webhook:*",
       "amd64OnlyVersions": [
-        "prod_20210714.1",
```
Comment (chengliangli0918) [inline vhdbuilder/packer/components.json:396]: prod_20210909.1 was not used in RP for a while 

{{- else if eq .component "azure-policy-webhook" -}}
  {{- if semverCompare ">=1.18.0" .version -}}prod_20211230.1
  {{- else -}}prod_20210714.1
  {{- end -}}

**Review by ganeshkumar5699 — APPROVED** [review]
Lgtm. Thanks for proactively helping us in reducing unused images!

**Review by anujmaheshwari1 — APPROVED** [review]

**Pull request merged**